heat/core/operations.py

```diff
@@ -105,7 +115,7 @@ def clip(a, a_min, a_max, out=None):
         raise ValueError('either a_min or a_max must be set')
 
     if out is None:
-        return tensor.tensor(a._tensor__array.clamp(a_min, a_max), a.shape, a.dtype, a.split, _copy(a._tensor__comm))
+        return tensor.tensor(a._tensor__array.clamp(a_min, a_max), a.shape, a.dtype, a.split, a.comm)
```

Comment:
I think we should spend setters for tensor attributes, i.e. allowing a.array.clamp(..
in this case and other accordingly